CTFd/auth.py

```diff
@@ -194,13 +201,23 @@ def register():
                     return redirect(url_for('auth.confirm'))
                 else:  # Don't care about confirming users
                     if config.can_send_mail():  # We want to notify the user that they have registered.
-                        email.sendmail(
-                            request.form['email'],
-                            "You've successfully registered for {}".format(get_config('ctf_name'))
-                        )
+                        if auto_create:
+                            msg = render_template(
+                                'email_password.html',
+                                name=name, password=password
+                            )
+                            email.sendmail(email_address, msg, 'html')
+                        else:
+                            email.sendmail(
+                                email_address,
+                                "You've successfully registered for {}".format(get_config('ctf_name'))
```

Comment:
If I understand correctly, this is the case when users are registered manually as opposed to those registered via landing page. Maybe we could announce to that when registering this way, they won't be eligible pro the prizes.